defender/downloaded/Contract_cToken.js

```diff
@@ -3,9 +3,24 @@ const axios = require('axios');
 const axiosRetry = require('axios-retry');
 const ethers = require('ethers');
 
+// function to calculate the delay until the next request attempt
+// returns a value specified in milliseconds
+function retryDelayFunc(retryCount) {
+  // 300 seconds total in Autotask execution to perform retries
+  // #   time   attempt
+  // 0 -   0s - initial request
+  // 1 -  40s - first retry (40s delay from initial request)
+  // 2 - 120s - second retry (80s delay from first retry)
+  // 3 - 280s - third retry (160s delay from second retry)
+  // this leaves 20s for the rest of the Autotask to execute, plus
+  // whatever time each request takes
+  const delay = (retryCount**2)*40*1000;
+  return delay;
+}
+
 axiosRetry(axios, {
   retries: 3,
-  retryDelay: axiosRetry.exponentialDelay,
+  retryDelay: retryDelayFunc,
```

Comment:
For the custom `retryDelayFunc()`, are the attempted delay backoffs matching the comments provided? 

With the way the function is currently written, would it look something like this:

```
first retry delay = (1**2)*40*1000 = 40 seconds
second retry delay = (2**2)*40*1000 = 160 seconds
third retry delay = (3**2)*40*1000 = 360 seconds
```

Reply:
You're absolutely right. Fixed!